Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -69,9 +69,12 @@ public class Chat {
     /// An observable object representing the current state of the channel.
     @MainActor public lazy var state: ChatState = stateBuilder.build()
     
-    /// Fetches the state from the server and updates the local store.
+    /// Fetches the most recent state from the server and updates the local store.
     ///
-    /// - Important: Loaded messages in ``ChatState.messages`` is reset to a batch of most recent messages.
+    /// - Important: Loaded messages in ``ChatState.messages`` are reset.
+    /// - Note: When watching is enabled for the channel, then channel updates are delivered
+    ///  through websocket events and there is no need to call ``get(watch:)`` for fetching
+    ///  the latest state multiple times during the app's lifetime.
```

Comment:
Currently we reset `messages` when calling `get(watch:)`. ChatState also has other properties:
```swift
 @Published public private(set) var members = StreamCollection<ChatChannelMember>([])
@Published public internal(set) var watchers = StreamCollection<ChatUser>([])
```

For consistency reason it probably makes sense that we reset these as well. WDYT?

Reply:
yes, makes sense

Reply:
I'll do that part in a separate PR since it needs some special handling for the member list part in Chat and it would be better to review that in isolation.